locales/de.json

```diff
@@ -180,7 +180,6 @@
 	"An error occurred while publishing your message.": "Ein Fehler trat auf während deine Nachricht publiziert wurde.",
 	"Missing message": "Nachricht fehlt",
 	" via ": " über ",
-	"Follow Back": "Zurück Folgen",
```

Comment:
:man_shrugging: I think this was an automated change, but I have no idea why it would be made. I've reverted all of the `locale` directory changes in 198b91c, making only one change to remove an invalid JSON character.